examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -188,27 +191,28 @@ class _SelectableTravelDestinationItemState extends State<SelectableTravelDestin
                       _isSelected = !_isSelected;
                     });
                   },
-                  splashColor: Theme.of(context).colorScheme.primary.withAlpha(30),
+                  splashColor: colorScheme.primary.withAlpha(30),
```

Comment:
should we be using withOpacity instead of withAlpha since material usually defines opacity as % ?

Reply:
Good call, updated the values themselves as well.